recipes/shot-scraper/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "shot-scraper" %}
+{% set version = "1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/shot-scraper-{{ version }}.tar.gz
+  sha256: 84fccec2f4d0005b3d210464abfbb0fa91efe593f95d4dd99fad3948bf2e74f0
+
+build:
+  entry_points:
+    - shot-scraper = shot_scraper.cli:cli
+  # Note Playright is not available for Win
+  skip: true  # [win]
```

Comment:
It should probably be a notch recipe

It is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows.

Reply:
reverted to NoArch